### PR TITLE
Changes for discussion to the PR Template

### PR DESCRIPTION
**Pull request by christothes**
This PR is intended for discussion purposes.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (tg-msft) [inline .github/PULL_REQUEST_TEMPLATE.md:10]:
```suggestion
- Leaving [suggested changes](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request) is welcomed, but please never commit them for a PR you did not create.
```

**Review by tg-msft — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (christothes) [inline .github/PULL_REQUEST_TEMPLATE.md:5]: This was distilled from comments during our team discussion today

**Review by christothes — COMMENTED** [review]
The

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (Mohit-Chakraborty) [inline .github/PULL_REQUEST_TEMPLATE.md:12]: 'Request Changes' in quotes perhaps to indicate it is a particular thing?

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (jsquire) [inline .github/PULL_REQUEST_TEMPLATE.md:3]:
```suggestion
Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.
```

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (jsquire) [inline .github/PULL_REQUEST_TEMPLATE.md:1]:
```suggestion
# Contributing to the Azure SDK
```

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (jsquire) [inline .github/PULL_REQUEST_TEMPLATE.md:16]:
```suggestion
- If you add  significant changes the PR after it has been marked approved, please confirm with reviewers that they still approve before merging.
```

**Review by jsquire — APPROVED** [review]

**Review by annelo-msft — COMMENTED** [review]
It would be great to get input on this from the Azure SDK .NET Devs in Shanghai as well.  I'll add them as reviewers.

**Comment by annelo-msft** [comment]
@m-nash could you add .NET collaborators from the management plane team?

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (AlexGhiondea) [inline .github/PULL_REQUEST_TEMPLATE.md:7]: This calls out all the things one shouldn't do... maybe we should also add things around approvals -- when should one approve? Something along the line of requesting approval from the area owner?

Comment (christothes) [reply .github/PULL_REQUEST_TEMPLATE.md:7]: >  Something along the line of requesting approval from the area owner?

Are there scenarios where this wouldn't be automatically done via codeowners?

Comment (AlexGhiondea) [reply .github/PULL_REQUEST_TEMPLATE.md:7]: I don't think there is a GitHub way to block on area owners. I know that Fabric Bot has an awesome PR flow where you can request (and the bot will block) PRs from certain people but we're not using that yet.

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (heaths) [reply .github/PULL_REQUEST_TEMPLATE.md:5]: I think all this text would be best in CONTRIBUTING.md. We could link to the bookmark directly.

Repeated studies show people don't like reading large blocks of text. I suspect - after seeing it more often in repos with larger template - people just `Ctrl+A + Del` when they see large templates like this with descriptive text just so they can write what they need. For example, have you ever seen an Azure/azure-rest-api-specs PR template filled out?

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (heaths) [inline .github/PULL_REQUEST_TEMPLATE.md:10]:
```suggestion
- Leaving [suggested changes](https://docs.github.com/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request) is welcomed, but please never commit them for a PR you did not create.
```

Just like with docs.microsoft.com links, do not include language when the site would otherwise use the browser's `Accept-Language` header to pick the best for the user.

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (heaths) [inline .github/PULL_REQUEST_TEMPLATE.md:12]: This really depends on the target audience. Not only is "request changes" implied if you don't have merge permissions, it may not be obvious to some contributors that we are indeed waiting for changes before we decide to merge.

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (christothes) [reply .github/PULL_REQUEST_TEMPLATE.md:5]: It's a good discussion. Arguably we should just anchor link to a few key parts, this being one of them.

**Comment by AlexGhiondea** [comment]
I like where we're landing -- I am wondering if we should put this into contributing.md as @heaths was mentioning and link to it from here? Thoughts?

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (christothes) [reply .github/PULL_REQUEST_TEMPLATE.md]: Moved it to the main CONTRIBUTING.md

**Comment by heaths** [comment]
I'd also like to see us try to come to some agreement about sign-offs. I know a lot of people like using "Add single comment" because they don't trust that uncommitted PR comments are saved. These appear as if someone made a single comment and resolved their PR review as merely "Comment". But how do PR authors know when they are done? They should either approve or request changes as their last review so we know when they are done. Otherwise, we're left here waiting a while to see if more come in, or have to ping them over Teams (if available) to ask and it becomes a big waste of time.

**Comment by christothes** [comment]
> I'd also like to see us try to come to some agreement about sign-offs. I know a lot of people like using "Add single comment" because they don't trust that uncommitted PR comments are saved. These appear as if someone made a single comment and resolved their PR review as merely "Comment". But how do PR authors know when they are done? They should either approve or request changes as their last review so we know when they are done. Otherwise, we're left here waiting a while to see if more come in, or have to ping them over Teams (if available) to ask and it becomes a big waste of time.

Did you leave this as a single comment ironically? 😄 

I agree, I think general PR comments should be avoided unless they are due to a general comment that applies to the whole PR like "this gemeral approach should be reconsidered".

**Comment by heaths** [comment]
> Did you leave this as a single comment ironically? 😄

Let's just say it was a reminder. 😉

**Comment by heaths** [comment]
> I agree, I think general PR comments should be avoided unless they are due to a general comment that applies to the whole PR like "this gemeral approach should be reconsidered".

Perhaps, but I've used this a few times where I didn't want to sign off one way or another because I was an area expert, but happened to notice a PR (or was asked to review) and had a few opinions. Moreover, I think just some way to signal "I'm done reviewing for now" is helpful to PR authors.

**Review by tg-msft — APPROVED** [review]

**Pull request merged**
